CollAction/ValidationAttributes/DateWithinMonthsAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace CollAction.ValidationAttributes
+{
+    public class DateWithinMonthsAttribute : ValidationAttribute
+    {
+        private readonly int _months;
+
+        public String OfDateProperty { get; set; }
+
+        public DateWithinMonthsAttribute(int months)
```

Comment:
To indicate a difference in dates, it's better and more general to use a timespan: https://msdn.microsoft.com/en-us/library/system.timespan(v=vs.110).aspx